server/routes/createGoal/createGoalsController.ts

```diff
@@ -1,23 +1,27 @@
 import type { RequestHandler } from 'express'
-import type { NewGoal } from 'compositeForms'
 import moment from 'moment'
+import type { CreateGoalsForm } from 'forms'
 import CreateGoalsView from './createGoalsView'
 import futureGoalTargetDateCalculator from '../futureGoalTargetDateCalculator'
 
 export default class CreateGoalsController {
   constructor() {}
 
-  // TODO: RR-734 - Create controller and view classes new create goal journey
   getCreateGoalsView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
 
-    req.session.newGoals = req.session.newGoals || []
+    req.session.newGoalsForm = req.session.newGoalsForm || []
 
-    if (!req.session.newGoal?.createGoalForm) {
-      req.session.newGoal = {
-        createGoalForm: { prisonNumber },
-      } as NewGoal
+    if (!req.session.newGoalsForm?.createGoalsForm) {
```

Comment:
Ah, the nested property `createGoalsForm` here is wrong - I can see where you got it from (from what we do on the current journey; but that is because there **is** a nested property in that case)
I think this line needs to be:
```
if (!req.session.createGoalsForm) {
```
 (on the basis that you've renamed the property re: my comment above)